client/components/Mentions.vue

```diff
@@ -211,14 +211,20 @@ export default {
 				this.isOpen = false;
 			}
 		},
-		openPopup() {
+		togglePopup() {
 			this.isOpen = !this.isOpen;
 
 			if (this.isOpen) {
 				this.isLoading = true;
 				socket.emit("mentions:get");
+				eventbus.on("escapekey", this.closePopup);
+			} else {
+				eventbus.off("escapekey", this.closePopup);
```

Comment:
We should to this on destruction as well. Hot reloading can cause destruction at unusual times but I bet there's non-dev-server ways for this component to be destroyed without `togglePopup` being called first as well.